File: SLAManager/SLAManager/EntityFramework/Models/SlaMetricViolation.cs
using System;
using System.Collections.Generic;

namespace SLAManagerdata.Models;

public partial class SlaMetricViolation
{
    public int Id { get; set; }

    public int? IdSla { get; set; }

    public string? Violation { get; set; }

    public string? Action { get; set; }

    public string? Code { get; set; }

    public string? Controller { get; set; }

    public string? Endpoint { get; set; }

    public string? Instance { get; set; }

    public string? Job { get; set; }

    public string? Method { get; set; }

    public DateTime? Datetime { get; set; }

    public double? MisuredValue { get; set; }

    public double? DesiredValue { get; set; }
}

File: SlaManagerService/app/Model/SlaMetricViolation.py
from datetime import datetime


class SlaMetricViolation:
    def __init__(self, IdSla=None, Violation=None,Action=None,Code=None,Controller=None,Endpoint=None,Instance=None,Job=None,Method=None,FromDesiredValue=None,ToDesiredValue=None,MisuredValue=None,Metric=None,MetricDescription=None):
        self.IdSla = IdSla
        self.Violation = Violation
        self.FromDesiredValue = FromDesiredValue
        self.ToDesiredValue = ToDesiredValue
        self.MisuredValue = MisuredValue 
        self.Datetime = datetime.utcnow()
        self.Metric = Metric
        self.MetricDescription = MetricDescription
        self.Action = Action
        self.Code = Code
        self.Controller = Controller
        self.Endpoint = Endpoint
        self.Instance = Instance
        self.Job = Job
        self.Method = Method